lib/active_model/serializer/railtie.rb

```diff
@@ -19,5 +19,7 @@ class Railtie < Rails::Railtie
       app.load_generators
       require 'generators/serializer/resource_override'
     end
+
+    ActionController::TestCase.send(:include, ActiveModelSerializers::Test::Schema)
```

Comment:
tl;dr let's verify if we can load AMS in production mode without the conditional

There's an issue about this in the ams issues.. I can go find it, but if you eager load the test **framework** in non-test, it will blow up trying to require minitest among other things.  

The difference here and the example links (thanks! I love how you research this stuff), is that e.g. we're patching an existing testcase, not writing our own. 

e.g. see [`class ActiveJobTestCaseTest < ActiveJob::TestCase`](https://github.com/rails/rails/blob/7f18ea14c893cb5c9f04d4fda9661126758332b5/activejob/test/cases/test_case_test.rb)

[ActiveJob::TestHelper](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/activejob/lib/active_job/test_helper.rb) has no explicit dependency on a test framework being loaded.  

[`ActiveSupport::TestCase`](https://github.com/rails/rails/blob/d7add5352e7c8b31850431bb93acb56d93c5dc64/activejob/lib/active_job/test_case.rb) does inherit from `ActiveSupport::TestCase`, but it is only ever loaded in test when subclassing it.

I think I verified this in console. Here's one of the issues I was thinking of https://github.com/rails-api/active_model_serializers/issues/814